master/docs/manual/configuration/workers-openstack.rst

```diff
@@ -52,6 +52,8 @@ These are the same details set in either environment variables or passed as opti
 
 ``flavor``
     The flavor ID to use for the instance.
+    A callable may instead be passed.
+    It will be passed the list of available flavors and must return the flavor to use.
```

Comment:
It's unclear what type of the flavor is.

`It will be passed the list of available flavors as a list of strings and must return the flavor to use as string.`